EndlessClient/Controllers/LoginController.cs

```diff
@@ -229,6 +230,15 @@ private async Task<bool> SafeGetFile(Func<Task> operation)
             return await op.Invoke();
         }
 
+        private void ClearChat()
+        {
+            foreach(var chat in _chatRepository.AllChat.Values)
```

Comment:
Do this via linq - `foreach (var chat in _chatRepository.AllChat.Values.Where(x => x.Count > 0)) { chat.Clear(); }`. You could also just do it unconditionally since there's only 6 or so chat collections anyway

Reply:
Removed condition check :)